src/css/_widget-color.react.css

```diff
@@ -0,0 +1,163 @@
+/* React: Widget Color */
+
+.drag {
+    background-color: var(--ui-component-color);
+    cursor: move;
+    height: 20px;
+    justify-content: flex-end;
+    display: flex;
+}
+
+.widget-exit {
+    padding: 0;
+    background-color: transparent;
+    outline: none;
+    border-color: transparent;
+    border: transparent;
+    padding-right: 14px;
+    text-align: center;
+    justify-content: center;
+    display: flex;
+}
+
+.widget-exit i {
+    font-size: 8px;
+    font-weight: 600;
+    color: white;
+}
+
+.widget-exit i:hover {
+    color: yellow;
+}
+
+/* color-picker.react */
+
+.widget-colorpicker {
+    box-sizing: border-box;
+    border: 1px solid rgba(255, 255, 255, 0.15);
+    border-radius: 50%;
+    max-width: 14px;
+    max-height: 14px;
+    width: 1em;
+    height: 1em;
+    cursor: pointer;
+    vertical-align: middle;
+    margin-top: -0.15em;
+}
+
+/* color-picker-box.react */
+
+.widget-color-box {
+    width: 220px;
+    padding: 14px;
+    box-sizing: initial;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.15);
+    background-color: var(--ui-element-color);
+}
+
+.widget-color-box .saturation {
+    width: 100%;
+    padding-bottom: 75%;
+    position: relative;
+    overflow: hidden;
+}
+
+.widget-color-box .saturation2 {
+    radius: 3px;
+    box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.15);
+}
+
+.widget-color-box .controls {
+    display: flex;
+}
+
+.widget-color-box .sliders {
+    padding: 4px 0;
+    flex: 1;
+}
+
+.widget-color-box .color {
+    width: 24px;
+    height: 24px;
+    position: relative;
+    margin-top: 4px;
+    margin-left: 4px;
+}
+
+.widget-color-box .activeColor {
+    width: 100%;
+    height: 100%;
+}
+
+.widget-color-box .hue {
+    position: relative;
+    height: 10px;
+    overflow: hidden;
+}
+
+.widget-color-box .hue2 {
+    radius: 2px;
+    box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.15);
+}
+
+.widget-color-box .alpha {
+    position: relative;
+    height: 10px;
+    margin-top: 4px;
+    overflow: hidden;
+}
+
+.widget-color-box .alpha2 {
+    box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.15);
+}
+
+/* color-picker-input-fields.react */
+
+.fields {
+    display: flex;
+    padding-top: 4px;
+}
+
+.fields .single {
+    flex: 1;
+    padding-left: 3px;
+}
+
+.fields .alpha3 {
+    flex: 1;
+    padding-left: 6px;
+}
+
+.fields .double {
+    flex: 2;
+}
+
+.widget-color-box input {
+    width: 80%;
+    padding: 4px 8% 3px;
+    border: none;
+    outline: none;
+    background-color: #55575a;
+    text-align: center;
+    font-size: 12px;
+    color: white;
+}
+
+.single div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.single span {
```

Comment:
Would it also be possible to prefix `.drag`, `.fields`, `.double`, `.single` or would that require too much work in changing the `react-color` component?